test/spec/fixtures/stack/healthcheck/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:latest
+
+ADD server.py .
+
+CMD ["python3", "server.py"]
```

Comment:
Why 🐍 ?

Reply:
First thing at hand. I can rewrite this in ruby using `WEBrick`...